tests/testthat/test-docnames.R

```diff
@@ -0,0 +1,12 @@
+context("test docnames")
+
+test_that("docnames always return names even if there aren't", {
+    
+    corp <- corpus(c('aaa', 'bbb', 'ccc'))
+    expect_equal(length(docnames(corp)), ndoc(corp))
+    
+    toks <- as.tokens(list('aaa', 'bbb', 'ccc'))
```

Comment:
We need to have some kind of integrity check for all the `corpus` and `tokens`. S4 could be one, but could be done in S3. The main point of corpus overhaul.